app/views/map/_peopleLeaflet.html.erb

```diff
@@ -10,10 +10,9 @@
  
   <p><i><small>
     <% if !@map_lat.nil? && !@map_lon.nil? %>
-      Current view based on your shared location. Click <a href='/profile'>here</a> to adjust.
+      <%= translation('map.index.current_view') %>
```

Comment:
Ah! I think this is failing while it's running in a rare and unexpected way, which we do sometimes with this type of partial template -- within our models. It's not how rails was designed, so it's not your fault!

```
Progress: |===================================================ERROR NodeSharedTest#test_that_NodeShared_can_be_used_to_convert_short_codes_like_[map:people:___:___]_into_maps_which_display_peoples_locations (15.88s)
Minitest::UnexpectedError:         ActionView::Template::Error: undefined method `translation' for #<#<Class:0x0000562fffed8930>:0x0000562ffe62def8>
            app/views/map/_peopleLeaflet.html.erb:15:in `_app_views_map__people_eaflet_html_erb__1509984826448044750_47382065653620'
            app/models/concerns/node_shared.rb:368:in `map_data_string'
            app/models/concerns/node_shared.rb:220:in `block in people_map'
            app/models/concerns/node_shared.rb:215:in `gsub'
            app/models/concerns/node_shared.rb:215:in `people_map'
            test/unit/node_shared_test.rb:168:in `block in <class:NodeSharedTest>'
```

We may need to use a more full reference to the `translate()` helper here, as it won't be pre-loaded and ready to use in the context of model (unit) code. Let me think...

Reply:
```suggestion
      <%= ActionController::Base.helpers.translation('map.index.current_view') %>
```